mypy_django_plugin/config.py

```diff
@@ -14,22 +15,24 @@
 (config)
 ...
 [mypy.plugins.django-stubs]
-django_settings_module = str (required)
+django_settings_module = str (default: DJANGO_SETTINGS_MODULE env var)
```

Comment:
```suggestion
django_settings_module = str (default: `os.getenv(DJANGO_SETTINGS_MODULE)`)
```

let's be consistent.